Extensions/activity_plus.js

```diff
@@ -1,5 +1,5 @@
 //* TITLE Activity+ **//
-//* VERSION 0.3 REV C **//
+//* VERSION 0.3.3 **//
```

Comment:
listen to @wolvan he wrote the code

On Mon, Dec 28, 2015 at 9:35 PM nepeat <notifications@github.com> wrote:

> In Extensions/activity_plus.js
> <https://github.com/new-xkit/XKit/pull/890#discussion_r48520216>:
>
> > @@ -1,5 +1,5 @@
> >  //* TITLE Activity+ **//
> > -//* VERSION 0.3 REV C **//
> > +//* VERSION 0.3.3 **//
>
> Rev C => 2 right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/new-xkit/XKit/pull/890/files#r48520216>.
>